lms/djangoapps/student_account/views.py

```diff
@@ -241,6 +241,8 @@ def update_context_for_enterprise(request, context):
     context = context.copy()
 
     sidebar_context = enterprise_sidebar_context(request)
+    enterprise_customer = enterprise_customer_for_request(request)
```

Comment:
We've already done this in `enterprise_sidebar_context`. No need to do it again.